doc/Caliptra.md

```diff
@@ -1240,48 +1238,12 @@ The PAUSER field of the APB interface will be used to encode device attributes f
 
 ### Mailbox Commands
 
-\[\*\] denotes commands that are implemented in ROM; all others are implemented only in mutable firmware. Note that once Caliptra firmware is executing, firmware may provide alternate implementations of the noted commands.
+The Caliptra mailbox commands are specified in the runtime firmware specification: https://github.com/chipsalliance/caliptra-sw/blob/main/runtime/README.md#maibox-commands
 
-*Table 12: Caliptra Mailbox Commands*
```

Comment:
Did we lose this table? Is this intended?

Reply:
Yes it is intended. I replaced with a link to the runtime spec which has the latest commands.